### ISSUE TITLE
Console error when "OSE Initialized" Hook is registered

### ISSUE THREAD

**Issue by anthonyronda** [vttred/Old-School-Essentials-Advanced-Fantasy-for-Foundry-VTT#12]
### What happened?

1. No effect. Just a console error when Foundry VTT is loading.

### Relevant Errors and Warnings

```shell
ReferenceError: Error thrown in hooked function '' for hook 'OSE Initialized'. OSRCB is not defined
    <anonymous> http://localhost:30000/modules/ose-advancedfantasytome/scripts/classData.js:71
    #call http://localhost:30000/scripts/foundry.js:730
    call http://localhost:30000/scripts/foundry.js:712
    <anonymous> http://localhost:30000/modules/ose-advancedfantasytome/scripts/init.js:14
    #call http://localhost:30000/scripts/foundry.js:730
    callAll http://localhost:30000/scripts/foundry.js:687
    initialize http://localhost:30000/scripts/foundry.js:8529
    <anonymous> http://localhost:30000/scripts/foundry.js:90096
    async* http://localhost:30000/scripts/foundry.js:90077
foundry.js:753:30
    onError http://localhost:30000/scripts/foundry.js:753
    #call http://localhost:30000/scripts/foundry.js:734
    call http://localhost:30000/scripts/foundry.js:712
    <anonymous> http://localhost:30000/modules/ose-advancedfantasytome/scripts/init.js:14
    #call http://localhost:30000/scripts/foundry.js:730
    callAll http://localhost:30000/scripts/foundry.js:687
    initialize http://localhost:30000/scripts/foundry.js:8529
    <anonymous> http://localhost:30000/scripts/foundry.js:90096
    <anonymous> http://localhost:30000/scripts/foundry.js:90077
```

### Foundry VTT Core Version

V11

### Old-School Essentials Core Version

1.9.1

### Please update your browser regularly. Which (updated) browsers are you seeing the problem on?

Firefox

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by anthonyronda** [comment]
This bug is already addressed and the fix will be available in Advanced Fantasy 1.0.2

**Comment by anthonyronda** [comment]
Released
